### ISSUE TITLE
Windows support

### ISSUE THREAD

**Issue by aberonni** [decosoftware/deco-ide#14]
Any chance this will be distributed on windows?

**Comment by gdowens** [comment]
We definitely want it and hope to work with the community to get it out as fast as possible. Since we are using Electron, this is "in theory" doable within a short amount of time.

**Comment by rmilesson** [comment]
I would love to take a look at this as soon as I'm more familiar with the project overall.

**Comment by namuol** [comment]
Related cross-platform issue: #16

**Comment by dmk23** [comment]
Is there an estimated timeframe yet?

**Comment by elPatricko** [comment]
Any Update?

**Comment by dabbott** [comment]
Nope. Actually, we kind of de-prioritized this. But happy to accept PRs.

**Comment by hiteshsahu** [comment]
@dabbott Issue #284 and #274 are the duplicates, you might want to close them (2 fewer things to care about ). Window version would be really appreciated, though.

**Comment by antoninadert** [comment]
Still waiting...

**Comment by bitabs** [comment]
Still waiting...

**Comment by namuol** [comment]
Sometimes, life and other responsibilities get in the way of the free, open-source contributions you want to see happen.

Instead of nagging generous developers (reminder: if you're not a paying customer, you're probably not entitled), your time could be better-spent _helping_ with this or other OSS projects with similar goals.

I don't want my own personal biases to influence the success of one project over another, but here are some similar projects/products that happen to have some OSS bits and work cross-platform:

- [Structor](https://github.com/ipselon/structor)
- [Compositor Lab](https://compositor.io/lab/)

Please be patient and friendly with developers who donate their time and hard work for your benefit. Thanks! 🙇
